transform/app/services/solr/utils.py

```diff
@@ -18,3 +18,8 @@ def ids_mapping(id_: int | str, col_name: str) -> str:
             return str(id_ + settings.CATALOGUE_IDS_INCREMENTOR)
         case _:
             return id_
+
+
+def get_default_headers() -> dict:
+    """Return default headers for Solr API requests."""
+    return {"Content-Type": "application/json"}
```

Comment:
`{"Accept": "application/json", "Content-Type": "application/json"}.`
Adding "Accept": "application/json" in the headers tells the server that your client expects a JSON response

Reply:
Changed